admin-dev/themes/new-theme/scss/components/_notifications.scss

```diff
@@ -31,6 +31,8 @@
 
   .dropdown-menu {
     min-width: 25rem;
+    // stylelint-disable-next-line
+    height: auto !important;
```

Comment:
Are we able to avoid that !important? Or is it coming from bootstrap with an !important?

Reply:
There is a wrong calculation for height from bootstrap (768 < screen_size < 1024)
It's an inline style, and there is no way to be fixed without `!important`

![Screenshot from 2022-01-10 13-14-53](https://user-images.githubusercontent.com/85633460/148772092-c61e5849-bb08-4f3f-8802-4cc4ecdb1345.png)

Reply:
@mparvazi are you on a fresh install of PrestaShop 8.0? I tried to reproduce but the notification dropdown was well displayed on my side no matter the screen side 😅

Reply:
@atomix It happens in new theme pages (e.g. Orders / Screen width: 1000px)
https://github.com/PrestaShop/PrestaShop/issues/26792#issuecomment-981706573

Reply:
@mparvazi it looks like a JS issue rather than a css one 🤔.

Reply:
@atomiix 
It comes from bootstrap core. A responsive content inside a dropdown. Seems bootstrap calculates height before content rendered completely. 
The main problem is that the page content change to mobile view after width is lower than 1024px, but `.mobile` class will be added to main div when content is lower than 768px.

Reply:
Problem is here:

admin-dev/themes/new-theme/js/nav_bar.ts

![Screenshot from 2022-01-14 17-46-52](https://user-images.githubusercontent.com/85633460/149561439-a63e2a56-5296-4e70-9862-cb70f38beaf9.png)

Reply:
@NeOMakinG 

What does this class do (.js-notifs_dropdown)?

Reply:
@mparvazi I think it's only a js selector, no css is using this class AFAIK.

Reply:
@atomx 
It adds an inline style to the dropdown element after user click to open notification dropdown (e.g height: 700px).
Btw bootstrap controls dropdown UI and  there is no need to add `height` rule.
I tested by removing this class (.js-notifs_dropdown) from line:30 and everything was ok.

Reply:
I think you should remove the js code associated to this class rather than removing the css class

Reply:
I agree with @atomiix, what happens if we delete the JS CSS code? Maybe this is the only problem there